third_party/tinyxml2/tinyxml2.cpp

```diff
@@ -103,7 +103,7 @@ distribution.
 #if defined(_WIN64)
 	#define TIXML_FSEEK _fseeki64
 	#define TIXML_FTELL _ftelli64
-#elif defined(__APPLE__) || (__FreeBSD__)
+#elif defined(__APPLE__) || (__FreeBSD__) || (__OpenBSD__)

```

Comment:
Shouldn't this be a PR to upstream? Now if people compile with `-DBUNDLE_DEPENDENCIES=OFF` (as package maintainers might want), their TinyXML-2 will use the wrong fseek/ftell.

Reply:
Oh, missed that this was thirdparty - @strikersh, are you able to send this to upstream tinyxml2?

Reply:
submitted it there, and added Net and DragonFly bsd too (fseeko64 isn't available on bsds)